deps/rabbitmq_prometheus/src/rabbit_prometheus_handler.erl

```diff
@@ -59,7 +59,7 @@ setup_metrics(Registry) ->
                   {labels, ["registry", "content_type"]},
                   {registry, Registry}],
     ScrapeEncodedSize = [{name, ?SCRAPE_ENCODED_SIZE},
-                         {help, "Scrape size, encoded"},
+                         {help, "Scrape size, encoded (deprecated)"},
```

Comment:
Now that we don't eagerly compress with `zlib:gzip/1` we don't know how large compressed responses actually are so we can't record this metric. I doubt anyone was relying on / measuring this but I suppose it's a breaking change, eh? Should we remove this rather than never recording it?

Reply:
Yeah, I think you can just delete it